api_types.ts

```diff
@@ -62,13 +62,13 @@ interface TypeMap {
   [Type.string]: string;
   [Type.object]: object;
   [Type.boolean]: boolean;
-  [Type.date]: Date;
+  [Type.date]: string;
   [Type.html]: string;
   [Type.image]: string;
 }
 
 export type PackFormulaValue = $Values<Omit<TypeMap, Type.object>> | PackFormulaValue[];
-export type PackFormulaResult = $Values<TypeMap> | PackFormulaResult[];
+export type PackFormulaResult = $Values<Omit<TypeMap, Type.date>> | PackFormulaResult[];
```

Comment:
Hmm, is this saying that if you try to use `makeFormula` to define a formula that returns `Type.date` it won't compile? Seems like up until now we technically supported that case, right?

If so, was hoping we wouldn't be disallowing that, and could somehow reconcile dates as parameters being strings while dates as return values still being Dates.

Reply:
Oh you're right, we changed `makeFormula()` to require declaring a primitive type as the result type (ValueType) and if you want to get a rich value like a date you have to use codaType (ValueHintType). So this is probably fine.

Can you just patch this in to `coda` and `packs` and make sure there aren't any TS errors with this change?